schema.graphql

```diff
@@ -7,6 +143,15 @@ type User @entity {
   firstTradeTimestamp: BigInt!
   "List of orders placed by this user"
   ordersPlaced: [Order!]! @derivedFrom(field: "owner")
+  "isSolver"
+  isSolver: Boolean
+  ######################################################################################################################
+  "solvedAmount" 
+  numberOfTrades: Int! # write a how to do it in the query or add the prop. 
+  solvedAmountEth: BigDecimal
+  solvedAmountUsd: BigDecimal
+  tradeAmountUsd: BigDecimal
+  tradeAmountEth: BigDecimal
```

Comment:
One pair is for solvers (`solvedAmount...`) and another for users (`tradeAmount...`), right?

I'm wondering whether it makes sense to have both in the same verb tense for consistency (`tradeAmount` -> `trade[d]Amount`) or leave as is